faker/providers/address/dk_DK/__init__.py

```diff
@@ -0,0 +1,107 @@
+from .. import Provider as AddressProvider
+
+
+class Provider(AddressProvider):
+
+    # Building numbers dont go higher than a 1000
+    building_number_formats = ('%##', '%#', '%')
+
+    street_name_formats = ('{{street_prefix}} {{street_suffix}}', )
+
+    street_address_formats = ('{{street_name}} {{building_number}}',)
+
+    street_prefixes = (
+        'Gade', 'Allé', 'Parkere', 'Baggade', 'Gyde', 'Motorvej',
+    )
+
+    street_suffixes = (
+        'Åbenrå', 'Adelgade', 'Lavendelstræde', 'Nygade', 'Pilestræde',
+        'Rigensgade', 'Silkegade', 'Toldbodgade', 'Blekingegade',
+        'Amalievej', 'Bülowsvej', 'Godthåbsvej', 'Howitzvej',
+        'Kingosgade', 'Martinsvej', 'Rolighedsvej', 'Schønbergsgade',
+        'Åboulevard', 'Ewaldsgade', 'Heinesgade', 'Kapelvej',
+    )
+
+    address_formats = ("{{street_address}}\n{{postcode}} {{city}}", )
+
+    # Postcode should be formatted as described in https://pastebin.com/NcLj96e7
```

Comment:
Is there a better source available than a random pastebin?

Reply:
The pastebin was mentioned in the issue, that's why I used it. But there's another reference, I will change it to that. I will also see why the tests are failing! Thank you!